vc_agent

```diff
@@ -10,6 +17,8 @@ export VC_AGENT_INSTALL_DIR=$VC_HOME/bin
 export VC_DRV_MANUAL_HOST_URI="$DATABASE_URL"
 export VC_HOSTNAME="${VC_HOSTNAME:-$(hostname)}"
 
+[ -f $VC_HOME/.aws/credentials ] || make_credentials
```

Comment:
I don't think there are any required perms for the creds file, but I'll set 'em to 0644 just in case (which is what the working credentials file on my system is).